src/neuroglancer/ui/graph_multicut.ts

```diff
@@ -387,6 +387,12 @@ export class GraphOperationLayerView extends Tab {
               segmentationState.rootSegmentsAfterEdit!.clear();
               segmentationState.rootSegments.add(splitRoots);
               segmentationState.rootSegmentsAfterEdit!.add(splitRoots);
+              
+              if (splitRoots.length === 1) {
```

Comment:
is splitRoots ever length of 2? Thats what I would've expected

Reply:
Here is my thought process:
If multicut mode is on and the split length is 2, that most likely means the path should not be regenerated, because either a tiny segment was split off or we have reached the end of the multicuts